Makefile

```diff
@@ -104,20 +166,20 @@ endif
 	# Check that the version output appears on a line of its own (the -x option to grep).
 	# Tests that the "git tag" makes it into the binary. Main point is to catch "-dirty" builds
 	@echo "Checking if the tag made it into the binary"
-	docker run --rm calico/libnetwork-plugin$(ARCHTAG) -v | grep -x $(VERSION) || (echo "Reported version:" `docker run --rm calico/libnetwork-plugin$(ARCHTAG) -v` "\nExpected version: $(VERSION)" && exit 1)
-	docker tag calico/libnetwork-plugin$(ARCHTAG) calico/libnetwork-plugin$(ARCHTAG):$(VERSION)
-	docker tag calico/libnetwork-plugin$(ARCHTAG) quay.io/calico/libnetwork-plugin$(ARCHTAG):$(VERSION)
-	docker tag calico/libnetwork-plugin$(ARCHTAG) quay.io/calico/libnetwork-plugin$(ARCHTAG):latest
+	docker run --rm $(CONTAINER_NAME) -v | grep -x $(VERSION) || (echo "Reported version:" `docker run --rm $(CONTAINER_NAME) -v` "\nExpected version: $(VERSION)" && exit 1)
+	docker tag $(CONTAINER_NAME) calico/libnetwork-plugin:$(VERSION)
```

Comment:
This should be `calico/libnetwork-plugin:$(VERSION)-$(ARCH)`, as should the next 2 lines have `-$(ARCH)`

Reply:
These need to add the "arch-less" tags with `amd64` as default. See [here](https://github.com/projectcalico/typha/blob/master/Makefile#L342) and [here](https://github.com/projectcalico/typha/blob/master/Makefile#L387) and [here](https://github.com/projectcalico/typha/blob/master/Makefile#L391) (and following lines) from the typha `Makefile`